lib/bandit/http1/adapter.ex

```diff
@@ -16,6 +16,7 @@ defmodule Bandit.HTTP1.Adapter do
             upgrade: nil
 
   alias ThousandIsland.Socket
+  alias Plug.Conn.Utils, as: PlugUtils
```

Comment:
For the one use of this let's just use it directly below?

Reply:
Yes, but credo was complaining, so i aliased it.
I added a `disable` for `AliasUsage` to silence the warning.